uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/entitlement.model.js

```diff
@@ -73,6 +73,32 @@ define([
             });
         },
 
+        /**
+         * Returns an array of leave entitlements which have a value greater
+         * than zero, allows for overuse, or allows accruals of leave requests.
+         *
+         * @param {Array} absenceTypes - An array of absence types.
+         * @param {Array} entitlements - An array of entitlements.
+         * @return {Array}
+         */
+        getLeaveEntitlements: function (absenceTypes, entitlements) {
```

Comment:
can you show me where else we would need the _exact_ output that this method would provide?

Reply:
Here: https://github.com/civicrm/civihr/blob/84fa3c37339a5824d1d060c26dba422866e7fcc1/uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/staff-leave-report.component.js#L263

But was not changed because it's not 100% what this code needs since it also overrides the absence type balance when there are no entitlements for it, so it needs some working.